docs/Documentation.md

```diff
@@ -257,11 +467,22 @@ Use a JSON configuration file based on a [schema](../src/Fantomas/schema.json) t
 
 A default configuration file would look like
 ```json
-{
+{  
   "IndentSpaceNum":4,
   "PageWidth":120,
   "SemicolonAtEndOfLine":false,
-  "SpaceBeforeArgument":true ,
+  "SpaceBeforeUnitArgumentInUppercaseFunctionCall":false,
+  "SpaceBeforeUnitArgumentInLowercaseFunctionCall":false,
+  "SpaceBeforeParenthesesArgumentInUppercaseFunctionCall":false,
+  "SpaceBeforeParenthesesArgumentInLowercaseFunctionCall":true ,
+  "SpaceBeforeUnitParameterInUppercaseFunctionDefinition":false,
+  "SpaceBeforeUnitParameterInLowercaseFunctionDefinition":false,
+  "SpaceBeforeParenthesesInUppercaseFunctionDefinition":false,
+  "SpaceBeforeParenthesesInLowercaseFunctionDefinition":true ,
+  "SpaceBeforeUnitParameterInUppercaseClassConstructor":false,
+  "SpaceBeforeUnitParameterInLowercaseClassConstructor":false,
+  "SpaceBeforeParenthesesParameterInUppercaseClassConstructor":false,
+  "SpaceBeforeParenthesesParameterInLowercaseClassConstructor":false,
```

Comment:
2 remarks:
- When the config setting says "functions" does it also mean methods? (just asking because I remember you opened a github issue in some MS docs repo where someone was advocating for defaults that differed depending if the invokation was on a function or a method)
- When the config setting says "class constructor" does it also mean the invokation of it?

Reply:
NOTE: you reduced ctor setting to one (SpaceBeforeClassConstructor) but here in the documentation there are still 4 ^